statcord/client.py

```diff
@@ -41,6 +50,11 @@ def __init__(self, bot: commands.Bot, statcord_key: str, custom_1: Callable = No
         self.logger = logging.getLogger("statcord")
         self.logger.setLevel(logging.WARNING)
 
+        # configuration
```

Comment:
Does anyone actually use this?

Reply:
I would. Since each cluster is on the same vps (just a different process), these values compound and show several times more than the actual value on statcord.

Reply:
perhaps then, a singular boolean argumentresource_stats would be better?